src/Pagination/components/Pagination.ts

```diff
@@ -4,18 +4,28 @@ import * as classNames from "classnames";
 import { PageButton, PageButtonProps } from "./PageButton";
 import { IconType, ItemType, PageStyleType, UpdateSourceType } from "../Pagination";
 import { PageNumberView, PageNumberViewProps } from "./PageNumberView";
+import { OptionProps, PageSizeSelect } from "./PageSizeSelect";
+import { PageSize } from "./PageSize";
 
 export interface PaginationProps {
     hideUnusedPaging: boolean;
     items: ItemType[];
     listViewSize: number;
-    offset: number;
+    pageSize: number;
     publishedOffset?: number;
     publishedPageNumber?: number;
     onClickAction: (offset: number, pageNumber: number) => void;
     getMessageStatus: (currentOffset: number, offset: number, maxPageSize: number) => string;
     pagingStyle: PageStyleType;
     updateSource?: UpdateSourceType;
+    pageSizeOnChange?: (OptionProps: OnChangeProps) => void;
+    pageSizeOptions?: OptionProps[];
+}
+
+export interface OnChangeProps {
```

Comment:
Another that could be reused. For largely duplicated interfaces, you could extract into a common file.